functions/__fzf_preview_file.fish

```diff
@@ -1,17 +1,26 @@
 # helper function for __fzf_search_current_dir
 function __fzf_preview_file --description "Prints a preview for the given file based on its file type." --argument-names file_path
-    set file_type (file --brief -i "$file_path")
+    if test -d "$file_path"
+        set file_type "d"
```

Comment:
does exa not tell you if the file in question is a directory?

Reply:
+Yes, in fact, this was done for ls specifically. With ls, you get a total block in first-line when using -a option, why -a? to get the file flag if the file in question is a directory. It was a choice between 2 if-else block and using sed + awk to get the file flag needed.
+In the case of exa, you have -d option, which beautiful turns quiet if its a file. 
+So, `exa -dl $file_path | cut -c 1` would suffice.